src/views/vehicleServiceDetails/vehicleServicepage.jsx

```diff
@@ -162,17 +163,21 @@ class vehicleServicepage extends Component {
       editScheduleModal: !prevState.editScheduleModal,
       scheduleID: schedule_id,
       serviceID: service_id,
-      requestedServiceDate: moment(
-        requested_service_date,
-        'DD/MM/YYYY HH:MM:SS'
-      ).format('YYYY/MM/DD HH:MM:SS '),
-      actualServiceDate: moment(
-        actual_service_date,
-        'DD/MM/YYYY HH:MM:SS'
-      ).format('YYYY/MM/DD HH:MM:SS '),
-      serviceOutDate: moment(service_out_date, 'DD/MM/YYYY HH:MM:SS').format(
-        'YYYY/MM/DD HH:MM:SS '
-      ),
+      requestedServiceDate: moment(requested_service_date).isValid()
+        ? moment(requested_service_date, 'DD/MM/YYYY HH:MM:SS').format(
```

Comment:
input and output gonna be same throughout the project, so pls make a const function to reuse

this avoids:
repeated instance of moment across all files,
if you wanna change the format, you can handle at one place